libindy/src/utils/crypto/hash/openssl.rs

```diff
@@ -2,56 +2,42 @@ extern crate openssl;
 
 use errors::prelude::*;
 use self::openssl::error::ErrorStack;
-use self::openssl::hash::{DigestBytes, hash as openssl_hash, Hasher, MessageDigest};
+use self::openssl::hash::{Hasher, MessageDigest};
 
 pub const HASHBYTES: usize = 32;
 
+pub const EMPTY_HASH_BYTES : [u8; 32] = [227, 176, 196, 66, 152, 252, 28, 20, 154, 251, 244, 200, 153, 111, 185, 36, 39, 174, 65, 228, 100, 155, 147, 76, 164, 149, 153, 27, 120, 82, 184, 85];
```

Comment:
Please, could you make this as a `lazy_static!` block, so it would not look like a magic number?

Reply:
How about adding a comment that these bytes are the same bytes as `openssl_hash(MessageDigest::sha256(), &[])` ?
I think lazy_static are for when you can't use const.